models/app_version_event.go

```diff
@@ -10,8 +10,9 @@ import (
 // AppVersionEvent ...
 type AppVersionEvent struct {
 	Record
-	Status string `json:"status"`
-	Text   string `json:"event_text" gorm:"column:event_text"`
+	Status         string `json:"status"`
+	Text           string `json:"event_text" gorm:"column:event_text"`
+	IsLogAvailable bool   `db:"is_log_available" json:"is_log_available"`
```

Comment:
I think there's no need for the `db` tag
```suggestion
	IsLogAvailable bool   `json:"is_log_available"`
```